cpu/nrf52/include/nrf802154.h

```diff
@@ -42,6 +42,23 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup drivers_nrf52_802154_conf  nrf802154 driver compile configuration
+ * @ingroup drivers_nrf52_802154
+ * @ingroup config
+ * @{
+ */
+
+/**
+ * @brief NRF802154 default CCA threshold value for CCACTRL register.
+ *
+ * @note  This value was copied from the Nordic reference driver configuration
+ */
+#ifndef CONFIG_NRF802154_CCA_THRESH_DEFAULT
+#define CONFIG_NRF802154_CCA_THRESH_DEFAULT 0x14
```

Comment:
Just for later reference, the reference configurationcan be found [here](https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver/blob/master/src/nrf_802154_config.h)